contracts/external/compound/CTokenInterface.sol

```diff
@@ -1,12 +1,14 @@
 pragma solidity 0.6.4;
 
-interface CTokenInterface {
-    function totalSupply() external view returns (uint256);
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
+
+interface CTokenInterface is IERC20 {
+    function totalSupply() external override view returns (uint256);
     function underlying() external view returns (address);
     function balanceOfUnderlying(address owner) external returns (uint256);
     function supplyRatePerBlock() external returns (uint256);
     function exchangeRateCurrent() external returns (uint256);
     function mint(uint256 mintAmount) external returns (uint256);
-    function balanceOf(address user) external view returns (uint256);
+    function balanceOf(address user) external override view returns (uint256);
```

Comment:
✂️

Reply:
IERC20 also has a balanceOf() so this has to be "override"